tests/qgis-projects/tests/test_cache.md

```diff
@@ -0,0 +1,13 @@
+This test is to check the cache.
+The test could be:
+- [ ] Config the tile cache as file cache
+- [ ] Create the the cache with the lizmap command line
```

Comment:
@drillinP Please write command lines to execute.

Reply:
yes